packit/local_project.py

```diff
@@ -512,7 +521,7 @@ def merge_pr(self, pr_id: Union[str, int]) -> None:
             f"({commit_sha})\t{target_branch.commit.summary}"
         )
         try:
-            self.git_repo.git.merge(f"pr-changes/{pr_id}")
+            self.git_repo.git.merge(pr_head)
```

Comment:
oh, i see why you did the `force=True` in the _fetch_ part… nice

Reply:
Correct, tests were failing without the force. We needed a jedi on this one.